src/view/InventoryView.java

```diff
@@ -128,6 +118,11 @@ public void keyPressed(KeyEvent e) {
             public void keyReleased(KeyEvent e) {
             }
         });
+
+        this.setLayout(new BoxLayout(this, BoxLayout.Y_AXIS));
+        this.add(title);
+        this.add(itemInfo);
+        this.add(inventoryInfo);
```

Comment:
Consider displaying inventoryInfo before the textbox itemInfo, in terms of UI experience might be better for user to see all their current information first before adding/deleting